deploy/controller.yaml

```diff
@@ -0,0 +1,75 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: datamon-csi-controller
+  namespace: dev
+spec:
+  serviceName: "datamon-csi-driver"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: datamon-csi-driver
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: datamon-csi-driver
+    spec:
+      serviceAccountName: datamon-csi-controller-sa
+      containers:
+        - name: csi-external-attacher
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-attacher:v0.4.1
+          args:
+            - "--v=4"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
```

Comment:
as a further addition, what about using Ivan's [`envexpand`](https://github.com/oneconcern/depmon/blob/master/hack/envexpand.go) `hack/` [as in](https://github.com/oneconcern/depmon/blob/master/k8s/dev-pod.template.yaml) [`depmon`](https://github.com/oneconcern/depmon/blob/master/hack/dev-env/create_dev_pod.sh) to do simple templating?

this `ADDRESS` and the `CSI_ENDPOINT` used below could all use the same template value, and the env variable could be removed in favor of inlining the template variable with the `args` item.  inlining into `args` in particular would make it clearer that the environment variable isn't meant to be used elsewhere (if this is in fact the case).

Reply:
Will revisit the yaml generation when integrating with Argo.. for now the controller yaml will not be undergoing changes as it will be deployed only once. The current scheme seems simple enough.